python/seldon_core/seldon_client.py

```diff
@@ -1272,7 +1273,7 @@ def get_token(
         token = response.json()["access_token"]
         return token
     else:
-        print("Failed to get token:" + response.text)
+        logger.debug("Failed to get token:" + response.text)
```

Comment:
This is out of the scope of this issue: does it make sense to keep maintaining OAuth code in the client?

Reply:
yeah I was thinking the exact same thing, I agree we should add an issue to remove it